libhb/qsv_common.c

```diff
@@ -1004,13 +1004,23 @@ int hb_qsv_full_path_is_enabled(hb_job_t *job)
 {
     static int device_check_completed = 0;
     static int device_check_succeded = 0;
+    int filter_count = 0;
+
+    if (job->list_filter && hb_list_count(job->list_filter))
```

Comment:
This can be simplified
```
filter_count = hb_list_count(job->list_filter);
```
hb_list_count does the null check for you and returns 0 if null.

The rest look ok

Reply:
Done!